src/router/router.tsx

```diff
@@ -75,15 +75,12 @@ export const protectedRoutes: FullRouteObject[] = [
     path: 'projects',
     label: i18n.t('menu.projects'),
     Icon: ProjectsIcon,
-    privileges: [Privileges.ViewPersonalProject, Privileges.ViewPersonalTask],
+    privileges: [Privileges.ViewPersonalProject],
```

Comment:
**[blocker]**

We should remove the privileges part from this parent "projects" path completely, otherwise users without the ViewPersonalProject privilege can't see their tasks (which will be essentially all vendors + some PMs).